collections/Tanner/MS_Tanner_106.xml

```diff
@@ -99,24 +99,24 @@
                         authority of the crown during the minority of Edward VI. (English)</note>
                   </msItem>
                   <msItem n="14" xml:id="MS_Tanner_106-item14">
-                     <locus from="14v">(fol. 14v)</locus>
+                     <locus from="14v" to="15r" type="inferredEnd">(fol. 14v)</locus>
                      <note> Order from Dr. William Tresham, subdean of Christ Church, Oxford, for the
                         payment of five marks to one T. Byllingsley for two standard candlesticks,
                         &amp;c. ; Dec. 23d 1553. (English)</note>
                   </msItem>
                   <msItem n="15" xml:id="MS_Tanner_106-item15">
-                     <locus from="15v">(fol. 15v)</locus>
+                     <locus from="15v" to="15" type="inferredEnd">(fol. 15v)</locus>
```

Comment:
does this sort of example occur much do you think? How easier would it be to fix?